### DISCUSSION TITLE
How do I get the live feed?

### DISCUSSION THREAD

**Discussion by gerroon** [toyxyz/ComfyUI_toyxyz_test_nodes#44] [Q&A]
Hi

I can capture images and load them in Comfy using , load webcam image but "capture webcam" does not  seem to capture any image. I was reading the readme up and down but I cant locate how that works.

thanks.

**Comment by toyxyz** [comment]
Did you use the LoadWebcamImage or the Capture Webcam node? 

The former should be used with the Webcam App. And the latter captures the webcam directly. 

![image](https://github.com/toyxyz/ComfyUI_toyxyz_test_nodes/assets/8006000/d1b515c4-2840-4bd2-b156-22fe08d575ac)

![image](https://github.com/toyxyz/ComfyUI_toyxyz_test_nodes/assets/8006000/bb9383d9-c7f6-462c-9183-6124b0162a50)

**Reply by gerroon** [reply]
I use "run.bat" to run the cam.py app. I am not sure if any other app is needed. Like I mentioned that app captures frames which I can use to the load webcam image node to load images. I thought that capture webcam node was able to stream video directly from the screenshots. Maybe I missunderstood the use case. It sounds like capture webcam is meant to be used with real webcam hardware?

**Reply by toyxyz** [reply]
> I use "run.bat" to run the cam.py app. I am not sure if any other app is needed. Like I mentioned that app captures frames which I can use to the load webcam image node to load images. I thought that capture webcam node was able to stream video directly from the screenshots. Maybe I missunderstood the use case. It sounds like capture webcam is meant to be used with real webcam hardware?

The Webcam app captures your webcam or screen and saves it as an image. You can select the desired capture target from the list.   
 LoadWebcamImage node has the ability to load the saved image. First, make sure that an image file is being saved.
